### ISSUE TITLE
Dice Detonator's seems broken

### ISSUE THREAD

**Issue by onetruepurple** [TerryCavanagh/diceydungeonsbeta#212]
Seems like it keeps track of the dice thrown *before* the spell was even obtained?

https://cdn.discordapp.com/attachments/552131784939405332/592311636048609280/unknown.png

**Comment by TerryCavanagh** [comment]
Yep, this is broken pretty badly, so the equipment never actually got enabled for launch. Might still fix it someday - it's still in the spreadsheets, so I'll transfer this one.
